docs/references/signed-message.md

```diff
@@ -0,0 +1,87 @@
+
+```js
+kaia.recoverFromMessage('0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', '0x61626364', '0x90824271750d7a09f90a76b6f8ec1e5e2afd31790fea9f43e26c120fef3152be46ad09c76f87bd6c495859fa37127754f1f0780180df53eda80034dac036b8d31b', 'latest')
+
+"0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a"
+```
+
+## EIP-191 Ethereum Signed Message
+
+In Ethereum and several EVM chains, it is widely used practice to prefix the messages with "Ethereum Signed Message" before signing them. For details, please refer to [EIP-191](https://eips.ethereum.org/EIPS/eip-191). Kaia nodes and SDKs support this feature. As with KIP-97, the application has to take care with replay protection when using EIP-191. Nevertheless, using EIP-191 ensures compatibility with other ecosystem tools and streamlines the message handling logic, without the need for EIP/KIP branches.
+
+EIP-191 signatures are supported by:
+- Ethereum SDKs (ethers.js, web3.js, web3j, web3py, viem, etc)
+- [kaia-sdk](https://github.com/kaiachain/kaia-sdk) suite (ethers-ext, web3js-ext, web3j-ext, web3py-ext), as they inherit the message signing features from their respective Ethereum SDKs ([see docs](../sdk))
+- In Kaia nodes since v1.0.1, [`eth_sign`](../json-rpc/eth/sign), [`kaia_sign`](../json-rpc/kaia/sign), [`personal_sign`](../json-rpc/personal/sign) [`personal_ecRecover`](../json-rpc/personal/ec-recover) RPCs
+- In Kaia nodes of all versions, [`kaia_recoverFromMessage`](../json-rpc/kaia/recover-from-message) RPC
+
+ethers.js example:
+
+```js
+const wallet = new ethers.Wallet("0x6397f5bfcef382017268d21294aed3b82d479b67323f94f7065d92a43643f20f");
+await wallet.signMessage("abcd");
+
+'0xe67ddbb12ad7c85a28b082bb3f159e637229454d34824bd96c0df38e49bf92d42167ffba7565855585de0c32407b0622b0b66fdfe7bd6566d4a19ca40b39ec631b'
+```
+
+RPC example:
+
+```js
+personal.importRawKey('6397f5bfcef382017268d21294aed3b82d479b67323f94f7065d92a43643f20f', 'pass')
+personal.unlockAccount('0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', 'pass')
+personal.sign('0x61626364', '0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', 'pass')
+
+"0xe67ddbb12ad7c85a28b082bb3f159e637229454d34824bd96c0df38e49bf92d42167ffba7565855585de0c32407b0622b0b66fdfe7bd6566d4a19ca40b39ec631b"
+```
+
+```js
+kaia.recoverFromMessage('0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a', '0x61626364', '0xe67ddbb12ad7c85a28b082bb3f159e637229454d34824bd96c0df38e49bf92d42167ffba7565855585de0c32407b0622b0b66fdfe7bd6566d4a19ca40b39ec631b', 'latest')
+
+"0xbc7d1abe33e6ec19ca873a3042a4dcf49149bc7a"
+```
+
+## EIP-712 Typed Structured Data
+
+While EIP-191 and KIP-97 were standards for signing a single string, EIP-712 is a standard for signing application data structured in JSON format. This format is more human-readable and efficient to process on EVM. For details, please refer to [EIP-712](https://eips.ethereum.org/EIPS/eip-712). Note that this standard cannot fully prevent replay attacks. As with other standards, applications must take care about replay protection mechanisms.
```

Comment:
It's better to mention how it works in kaia side.
In the KIP-97 and EIP-191 section, we noticed how kaia node supports from the standard perspective.

Reply:
AFAIK EIP-712 isn't supported by kaia node nor Kaikas. That's why I kept this section short. dApps can use MetaMask to receive EIP-712 signature anyways.

Reply:
Or I can just add MetaMask example (e.g. https://docs.metamask.io/wallet/reference/eth_signtypeddata_v4/, https://docs.metamask.io/wallet/how-to/sign-data/)

Reply:
@blukat29 Yes, then  It would be better to add the example with the short notice of not supported in kaia.

Reply:
Added Kaikas example, and MetaMask docs
https://github.com/kaiachain/kaia-docs/commit/c3b6f67dbc7f82620da37a02f742e408fd720f6c